### DISCUSSION TITLE
[FEAT] - Predefinied LXC containers

### DISCUSSION THREAD

**Discussion by wsadza** [tteck/Proxmox#4001] [Show and tell]
**TLDR**;
LXC container building automation

---

I recently came across this fantastic repository, and it brought back memories of my Proxmox days! 😄
Back then, I got curious about how Canonical creates its predefined LXC containers and discovered - [distrobuilder](https://linuxcontainers.org/distrobuilder/docs/latest/reference/source/)

With that in mind, I put together a semi-automated Ansible playbook to create LXC containers:
https://github.com/wsadza/ansible-role-lxc-distrobuilder

I don't known if you want to use it or not; But i known is better to post it here instead let it rot somewhere. 😅
